src/test/java/org/dataguardians/datasynth/DataGeneratorTest.java

```diff
@@ -10,7 +10,8 @@
 
 public class DataGeneratorTest {
 
-    private TokenProvider provider = new ApiKey("YOUR-API-KEY");
+    private TokenProvider provider = new ApiKey(System.getenv("OPENAI_API_KEY"));
```

Comment:
Oh the intent was to have a file or env based api key provider, but we could probably add it to the apikey class, but since this is all experimental I'm good with this.